### PR TITLE
[DOCS] Adds fix related to Ransonware Canary files 

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Fixes https://github.com/elastic/security-docs/issues/2685.

Preview [here](https://security-docs_2881.docs-preview.app.elstc.co/guide/en/security/master/release-notes-header-8.6.0.html#bug-fixes-8.6.0) - Summary added to the end of the bug fixes and enhancements section.

### CODE REVIEW

**Review by jdu2600 — APPROVED** [review]
LGTM.

Do we typically link to the related [security advisory](https://discuss.elastic.co/t/elastic-8-3-1-8-3-0-and-7-17-5-security-update/308613) in these notes?
And should the security advisory be updated with the feature re-enable version?

cc: @ferullo @jtnk

File: docs/release-notes/8.6.asciidoc [hunk 1]
```diff
@@ -96,3 +96,4 @@ There are no deprecations in 8.6.0.
 * Fixes a visual bug with the fullscreen view of rule preview results ({pull}146687[#146687]).
 * Fixes a visual bug with the fullscreen view of Osquery results ({pull}147076[#147076]).
 * Fixes a bug with the refresh indicator on the Rule details page ({pull}147806[#147806]).
+* Ransomware canary files were re-enabled.
```
Comment (jmikell821) [inline docs/release-notes/8.6.asciidoc:99]:
```suggestion
* Reenables ransomware canary files. 
```

**Review by benironside — APPROVED** [review]
Looks good, but definitely different since we don't have a PR to link here.

**Comment by nastasha-solomon** [comment]
@jdu2600 in the past, we haven't received requests or instructions to link to related security advisories but we can definitely start doing that if needed. We just need to know the page URL and associated bug fix, feature, or enhancement.

cc: @jmikell821 @benironside

**Review by jmikell821 — APPROVED** [review]

**Pull request merged**